doc/classes/TileMap.xml

```diff
@@ -258,6 +263,11 @@
 			</argument>
 			<description>
 				Returns the tilemap (grid-based) coordinates corresponding to the given local position.
+				To use this with an absolute position, first calculate the local position with [method Node2D.to_local]:
+				[codeblock]
+				var local_position = myTilemap.to_local(map_position)
```

Comment:
```suggestion
				var local_position = myTilemap.to_local(global_position)
```

I think this is a here?

Reply:
argh, indeed. fixed

Reply:
Maybe use ‘my_tilemap’ instead of ‘myTilemap’ for consistency and to follow Godot naming conventions?

Reply:
i am new to godot, the camelCase is very much in my muscle memory i am afraid :-)

fixed it to follow the naming conventions.